### PR TITLE
removing mineral ui from sermons

### PR DESCRIPTION
**Pull request by readeral**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,5 +88,8 @@
         }
       }
     ]
+  },
+  "dependencies": {
+    "@sanity/client": "^1.149.2"
```
Comment (BarryThePenguin) [inline package.json]: This should be a `devDependency`

Comment (readeral) [reply package.json]: I need to remove it entirely

File: packages/ppt-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "@emotion/styled": "^10.0",
+    "react": "^16.0"
+  },
+  "dependencies": {
+    "@theme-ui/components": "^0.3.1",
+    "polished": "^3.4.3",
+    "pptxgenjs": "^3.1.0",
+    "prop-types": "^15.6.0",
+    "react-global-configuration": "^1.3.0",
+    "theme-ui": "^0.3.1"
```
Comment (BarryThePenguin) [inline packages/ppt-generator/package.json]: This should be a `peerDependencies`

File: packages/ppt-generator/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "files": [
+    "dist"
+  ],
+  "peerDependencies": {
+    "@emotion/core": "^10.0",
+    "@emotion/styled": "^10.0",
+    "react": "^16.0"
+  },
+  "dependencies": {
+    "@theme-ui/components": "^0.3.1",
```
Comment (BarryThePenguin) [inline packages/ppt-generator/package.json]: `theme-ui` exports components now, so we don't need to install this

**Pull request merged**